### ISSUE TITLE
Staging Review finding: Fix heading structure on confirmation page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#135206]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** govcio-vebt
**Product:** Request for Reimbursement of Licensing or Certification Test Fees
**Feature:** VA Form 22-0803

## Findings details

**VA.gov Experience Standard - issue:** [User encounters a component or pattern that isn't considered a best practice.](https://design.va.gov/about/experience-standards/#user-encounters-components-and-patterns-that-meet-best-practices)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @allison0034 (Design), @jasonday (Accessibility)

### Description

The confirmation page has process list. The first step in the process list is a h2 but should be a h3.

Additionally, the "Regional Processing Office mailing addresses" is an H2, with child accordions that H2s. This heading structure may be confusing for assistive technology users.  

<details><summary>Link, screenshot or steps to recreate</summary>

![heading](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/118361-allison-4.png?raw=true)

<img width="1124" height="470" alt="heading structure" src="https://github.com/user-attachments/assets/7a832829-0238-44d0-b5b3-c3198926738b" />

<img width="1136" height="720" alt="heading structure" src="https://github.com/user-attachments/assets/86b32507-ecc0-47f3-ac43-ac86dfbd0877" />

</details>

## Recommended action

Change the heading to a h3.

Update the Eastern Region and Western Region headings to an H3. 

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG22/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** [Typography](https://design.va.gov/foundation/typography)
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
